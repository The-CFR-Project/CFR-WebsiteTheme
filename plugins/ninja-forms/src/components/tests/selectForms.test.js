import { render, cleanup } from '@testing-library/react';
import { globalParams } from './mockData';
import { SelectForms } from '../';

describe( 'submissions-select-forms', () => {
	afterEach( cleanup );
	it( 'Matches snapshot', () => {
		const data = {
			data: {
				globalParams,
			},
		};

		const setForms = jest.fn();

		const { container } = render(
			<SelectForms data={ data } setForms={ setForms } />
		);
		expect( container ).toMatchSnapshot();
	} );
} );
