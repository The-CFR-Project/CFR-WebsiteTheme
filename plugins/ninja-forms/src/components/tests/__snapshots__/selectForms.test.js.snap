// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`submissions-select-forms Matches snapshot 1`] = `
<div>
  <div
    class="components-base-control"
  >
    <div
      class="components-base-control__field"
    >
      <label
        class="components-visually-hidden"
        for="inspector-select-control-0"
      >
        Select Forms
      </label>
      <select
        class="components-select-control__input"
        id="inspector-select-control-0"
        multiple=""
      >
        <option
          value="10"
        >
          Contact Me
        </option>
        <option
          value="11"
        >
          Contact Me
        </option>
        <option
          value="12"
        >
          Submission Test 4
        </option>
      </select>
    </div>
  </div>
</div>
`;
